playwright/tests/rex-test/rex.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
+
+test('open rex page', async ({ page }) => {
+    await page.goto('https://staging.openstax.org/books/physics/pages/preface');
+    expect(page.url()).toBe('https://staging.openstax.org/books/physics/pages/preface');
```

Comment:
Should use the base URLs instead (down the line, at least...)

Reply:
will do in the next PR while starting a real test